diff --git a/node_modules/react-native-paper/src/components/Drawer/DrawerCollapsedItem.tsx b/node_modules/react-native-paper/src/components/Drawer/DrawerCollapsedItem.tsx
index 6d54faf..b4f16f7 100644
--- a/node_modules/react-native-paper/src/components/Drawer/DrawerCollapsedItem.tsx
+++ b/node_modules/react-native-paper/src/components/Drawer/DrawerCollapsedItem.tsx
@@ -244,7 +244,8 @@ DrawerCollapsedItem.displayName = 'Drawer.CollapsedItem';
 
 const styles = StyleSheet.create({
   wrapper: {
-    width: 80,
+    minWidth:80,
+    maxWidth:100,
     marginBottom: 12,
     minHeight: itemSize,
     alignItems: 'center',
@@ -267,7 +268,7 @@ const styles = StyleSheet.create({
     alignSelf: 'stretch',
   },
   label: {
-    marginHorizontal: 12,
+    marginHorizontal: 0,
     marginTop: 4,
     textAlign: 'center',
   },
